fieldservice_stock/models/stock_request_order.py

```diff
@@ -12,8 +12,7 @@ class StockRequestOrder(models.Model):
         index=True, copy=False)
 
     @api.onchange('direction', 'fsm_order_id')
-    def _onchange_location_id(self):
-        super()._onchange_location_id()
+    def _onchange_direction_order_id(self):
```

Comment:
Is this actually needing to be changed and the super() removed or just is it just needed to add the 'warehouse_id' to the onchange field list?

Reply:
That was one of the old debugging pieces we had I am not sure how that got merged instead of what we finalized. Thanks for catching, just updated the PR